scripts/process_wang.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+set -o errexit
+
+. ./scripts/utils.sh
+
+main() {
+    # Download and process data which describes cell locations, boundaries,
+    # and gene expression levels. Multiple JSON output files are produced:
+    # The files are redudant, but this reduces the processing that needs
+    # to be done on the client-side.
+
+    get_CLI_args "$@"
+
+    CSV_IN="$INPUT/mermaid.csv"
+    PNG_IN="$INPUT/mermaid.png"
+
+    CLI_ARGS="--csv_file $CSV_IN"
+    add_CLI_ARGS 'cells' 'mermaid'
+    add_CLI_ARGS 'molecules' 'mermaid'
+    add_CLI_ARGS 'images' 'mermaid'
+
+    echo "Download and process cells..."
+
+    if [ ! -e "$CSV_IN" ]
+    then
+        wget "$MERMAID_URL/data.csv.gz" -O "$CSV_IN.gz"
+        gunzip -df "$CSV_IN"
+    fi
+
+    [ -e "$PNG_IN" ] || \
+        wget "$MERMAID_URL/bg.png" -O "$PNG_IN"
+
+    IMAGE_OUT="$OUTPUT/mermaid.png"
+    if [ -e "$IMAGE_OUT" ]
+    then
+        echo "Skipping image -- output already exists: $IMAGE_OUT"
+    else
+        cp $PNG_IN $OUTPUT/mermaid.png
+    fi
+
+    TILE_BASE='mermaid.images'
+    TILE_PATH="$OUTPUT/$TILE_BASE"
+    if [ -e "$OUTPUT/mermaid.images/info.json" ]
```

Comment:
```suggestion
    if [ -e "$TILE_PATH/info.json" ]
```

This way it's immediately obvious where the error message which follows is coming from.